### DISCUSSION TITLE
Track numbers in file name with preceding zeros

### DISCUSSION THREAD

**Discussion by githubser** [audiamus/AaxAudioConverter#180] [Q&A]
Hi, 
First of all: Thanks for this great program! It is really one of the best and works fine. One litte question: I recently converted an Audible aax file with 348 chapters using the second option (Multiple files, by chapter). It results in files named (in german) "Kapitel 1 - ...", "Kapitel 2 - ..." etc. up to "Kapitel 348 - ...". 
Unfortunately my mp3 player in my car and also the windows Explorer has a problem to sort the files correctly, because they use alphabetical sorting. Is there a way to include preceding zeros to the track numbers in the file name, e.g. "Kapitel 001 - ...", "Kapitel 010 - ...", "Kapitel 100 - ...". Also the Track numbering within the mp3 files should be optionally with preceding zeros, because some players also use alphabetic sorting in track numbers. Would this be possible?

**Comment by audiamus** [comment]
The audio system in your car might support playlists (.m3u). That would be the easiest since all modes that split a book will create such a playlist ("Wiedergabeliste").

The other option is with explicit numerical prefix and leading zeros, to be activated in the Basic Settings, Chapter tab, See here for a recent reply in German: https://github.com/audiamus/AaxAudioConverter/issues/174
